datasources/windows/kubernetes/gke/kubernetes/telemetry/traces/traces.ps1

```diff
@@ -26,6 +26,11 @@ Invoke-Task 'Get-LogzioSpmToken' @{} 'Getting Logz.io SPM token' @($TracesFuncti
 Invoke-Task 'Build-EnableSpmHelmSet' @{} 'Building enable SPM Helm set' @($TracesFunctionsScript)
 # Build Logz.io SPM token Helm set
 Invoke-Task 'Build-LogzioSpmTokenHelmSet' @{SpmToken = $script:SpmToken} 'Building Logz.io SPM token Helm set' @($TracesFunctionsScript)
-
+# Get if service graph option was selected
+Invoke-Task 'Get-IsServiceGraphWasSelected' @{MetricsParams = $script:MetricsParams} 'Getting if service graph was selected' @($MetricsFunctionsScript)
+if ($script:IsServiceGraph) {
+    # Build enable metrics filter Helm set
```

Comment:
This comment is a copy-paste mistake? it supposed to be "build enable service graph"?